gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -79,6 +79,7 @@
  */
 @InternalExtensionOnly
 public final class InstantiatingGrpcChannelProvider implements TransportChannelProvider {
+  static final String DIRECT_PATH_ENV_VAR = "GOOGLE_CLOUD_ENABLE_DIRECT_PATH";
```

Comment:
private

Reply:
The new ones should be private, I agree, but for the old ones, I guess it will technically make it a breaknig change

Reply:
I made the new one private, and keep the old one as what it is (i.e., without adding private).